roles/ssp/handlers/main.yml

```diff
@@ -5,3 +5,12 @@
     name: "{{ ssp_webserver }}"
     state: restarted
   become: yes
+
+
+- name: Update symbolic link to SSP
+  file:
+    src: "{{ ssp_path }}/www/"
+    dest: "{{ ssp_path }}/../proxy-current"
```

Comment:
Introduce ansible var for SSP symbolic link, i.e. `ssp_www_path`.
This should default to `{{ ssp_path }}/../proxy-current"`

Reply:
Done.